subgraphs/uniswap-forks/src/mappings/masterChef/apeswap/reward.ts

```diff
@@ -1,8 +1,15 @@
 // import { log } from "@graphprotocol/graph-ts";
+<<<<<<<< HEAD:subgraphs/uniswap-forks/protocols/apeswap/src/mappings/masterchef/reward.ts
```

Comment:
merge conflict

Reply:
I actually needed to just delete the masterchef folder here. It was put back in when I deleted a commit per Oxbe1's comments. But I remembered I left out because of the directory structure update where this code is stored in the protocol specific folder.